utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/wifi_stations.lua

```diff
@@ -22,12 +37,34 @@ local function scrape()
           ifname = ifname,
           mac = mac,
         }
-        metric_wifi_station_signal(labels, station.signal)
+        if station.signal and station.signal ~= 0 then
+          metric_wifi_station_signal(labels, station.signal)
+        end
+        if station.inactive then
+          metric_wifi_station_inactive(labels, station.inactive)
+        end
+        if station.expected_throughput and station.expected_throughput ~= 0 then
+          metric_wifi_station_exp_thr(labels, station.expected_throughput)
+        end
+        if station.tx_rate and station.tx_rate ~= 0 then
+          metric_wifi_station_tx_bitrate(labels, station.tx_rate)
+        end
+        if station.rx_rate and station.rx_rate ~= 0 then
+          metric_wifi_station_rx_bitrate(labels, station.rx_rate)
+        end
         metric_wifi_station_tx_packets(labels, station.tx_packets)
         metric_wifi_station_rx_packets(labels, station.rx_packets)
+        if station.tx_bytes then
```

Comment:
are the if mandatory here, because if we have the number of packets, we should have the number of bytes
(either if is useless for bytes, or if is missing for packets)

Reply:
Well, yes. For this moment iwinfo.lua does not provide tx_bytes and rx_bytes.
A have a simple patch that makes it possible to fetch this parameters. And I plan to push this patch too.

Reply:
So, for this moment we need to use IF. May be later when my patch will be merged into mainstream iwinfo package, we can remove these IFs.

Reply:
Please see https://github.com/openwrt/openwrt/pull/1986 if you need some information about this patch.